### ISSUE TITLE
Unintended Print Statement on the 'ExistingCountries.search_fuzzy' function on '__init__.py' file

### ISSUE THREAD

**Issue by CollinsMunene** [pycountry/pycountry#226]
**Steps to Reproduce**
Call the pycountry.countries.search_fuzzy method with any country as a variable.
Observe the console output.

**Code Snippet**
```
      # Prio 2: exact matches on subdivision names
        match_subdivions = pycountry.Subdivisions.match(
            self=subdivisions, query=query
        )
        for candidate in match_subdivions:
            print(candidate) #this line should be remove
            add_result(candidate.country, 49)
```

**Expected Behavior**
The pycountry.countries.search_fuzzy method should not print anything to the console. Instead, it should handle results internally and return them to the caller.

**Actual Behavior**
The pycountry.countries.search_fuzzy method prints each candidate subdivision to the console, which results in unwanted output.

**Environment**
pycountry version: 23.12.11
Python version: 3.12.2
Operating System: macOS

**Additional Information**
This print statement can clutter the console output, especially when the library is used in a production environment or in scripts where clean output is crucial.

**Proposed Solution**
Remove or comment out the print(candidate) line in the pycountry.countries.search_fuzzy method or provide an option to disable this debug print.

Thank You

**Comment by mjvankampen** [comment]
https://github.com/pycountry/pycountry/issues/218
https://github.com/pycountry/pycountry/issues/200

same :-)

**Comment by zware** [comment]
This is a duplicate of #200, fixed in #201 for the next release.

**Comment by CollinsMunene** [comment]
Great
